### DISCUSSION TITLE
GitHub Copilot

### DISCUSSION THREAD

**Discussion by luthors** [rramosp/ai4eng.v1#50] [00 - General]
A modo de discusión les pregunto:  ¿Qué opinan de las nuevas tendencias de IA como github Copilot, que visión tienen a futuro acerca de estas?

**Comment by Felipe-RA** [comment]
La verdad es que es bastante interesante,  todo el código abierto en github era una oportunidad demasiado buena como para no ser aprovechada. Estoy a la espera de que lo abran  al público general para probarlo yo mismo.

El canal de divulgación de **dotcsv** hizo un vídeo hablando un poco sobre el tema de la seguridad laboral de los programadores y el impacto que puede tener Github Copilot ([cosa que también mencionan en su FAQ sobre IA responsable los de Github](https://copilot.github.com/)

Aquí el vídeo del divulgador: https://www.youtube.com/watch?v=uCpE6z999Uk

**Comment by AxelReboredo** [comment]
Me encanta que la tecnologia evolucione asi de rapido. Creo que Github Copilot es una herramienta que se va  a usar un monton. Escuche que varias personas piensan que este programa le va a sacar el protagonismo a los programadores. A mi me gusta pensarlo como una herramienta de apredizaje y ayuda.
